CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+Contributing Translations to Search.gov
```

Comment:
Search.gov's `CONTRIBUTING.md` page should apply to any changes, not necessarily just translations. I suggest we:
- change the primary heading to `Contributing to Search.gov` instead of `Contributing Translations to Search.gov`
- nest the translations-specific guidance under a `Translations` heading (or just move all of that copy, including the note about the Punchcard transition) under `Contributing to localization files`